server/controllers/paymentMethod.js

```diff
@@ -0,0 +1,32 @@
+const stripe = require('../utils/stripe');
+const asyncHandler = require("express-async-handler");
+
+
+// @route POST /payment_methods
+// @desc create payment method
+// @access Public
+exports.createPaymentMethod = asyncHandler(async (req, res, next) => {
+  paymentMethod = stripe.createPaymentMethod(req.body);
```

Comment:
`const paymentMethod = . . .`

Reply:
you can see some of this was done at 1-3am lol

Reply:
Ahh 😂, no problem.